src/Engine/UnitActionLifecycles/LoadLifecycle.java

```diff
@@ -99,6 +99,9 @@ public GameEventQueue getEvents(MapMaster gameMap)
         }
       }
 
+      // Make it so the passenger actually executes the move
+      movePath.snipCollision(gameMap, passenger);
```

Comment:
The seaplanes now do a move animation to the space adjacent to the transport instead of onto it.

Reply:
Right. Because otherwise there was *no* animation IIRC, since the move failed to validate to execute.
I don't like having validation failures as part of the happy path.

Reply:
There was an animation, and it went from unit to transport.
There was _also_ a `WARNING! Unable to move Infantry to (6, 6)`

Reply:
But the current implementation can also trigger that warning by moving over a unit onto a transport.

Reply:
Fair enough.